card/mod/account/set/right/account.rb

```diff
@@ -8,6 +8,70 @@
 card_accessor :status
 card_accessor :token
 
+#### ON CREATE
+
```

Comment:
this is mostly reordering to what I'm promoting as our default order (events on top, then card methods, then  formats (within which we go views then methods)